content/docs/rfcs/12/README.md

```diff
@@ -61,77 +63,150 @@ The following are not considered as part of the adversarial model:
 ## Protobuf
```

Comment:
I prefer the term "Wire format". It is more generic.

Reply:
I agree! Note that this PR specifically avoids format/heading/layout changes so that only the suggested wire format change can be reviewed. Once done, next step would be to adhere to the template and add theory, implementation, etc. sections.